src/olympia/amo/tests/test_utils_.py

```diff
@@ -172,3 +175,19 @@ def bar(self):
     del foo.bar
     assert foo.bar == 'original value'
     assert callme.call_count == 1
+
+
+@pytest.mark.parametrize('lang', settings.AMO_LANGUAGES)
+def test_get_locale_from_lang(lang):
+    """Make sure all languages in settings.AMO_LANGUAGES can be resolved."""
+    locale = get_locale_from_lang(lang)
+
+    assert isinstance(locale, Locale)
+    assert locale.language == lang[:2]
```

Comment:
This did not pass without to the changes from this PR... even with the previous babel version (`locale.language` returns the language with underscore and territory for all "long" languages, even `en-US`). 

I'm not sure what to make of that. Does it mean that `get_locale_from_lang` has always been broken, but the new Babel started to make the issue more visible ?